.gitignore

```diff
@@ -1,7 +1,8 @@
+npm
```

Comment:
why ? O.o

Reply:
The question is "why not" or "why has it been missing until now"? The JS script `npm` (in analogy to the PHP script `composer`) is the package manager which is either installed locally or as part of the OS. In any case it is _not_ part of Lychee.